### DISCUSSION TITLE
"Lotus chain export..." crashes occasionally.

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9190] [Storage Provider]
When executing the command "Lotus chain export -- recent states = XXX -- skip old msgs my snapshot. Car", the "Lotus daemon" will crash occasionally. How to solve this problem?

**Comment by Reiers** [comment]
Like the Docs states: This is a resource demanding task for the node. It may take up to 8 hours to create the CAR-snapshot, and the process will use around 70GB of RAM.

https://lotus.filecoin.io/lotus/manage/chain-management/#creating-a-snapshot

And thats exporting -- recent states = 2000, **if you add more you might even need more resources.** 

Whats the hardware specs for your node?

**Comment by ChaiWenyu1983** [comment]
cpu：amd3960x
ram：256G
gpu：3970.

**Reply by Reiers** [reply]
How many recent states did you export?
